content/c-sharp/concepts/strings/terms/toupper/toupper.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: '.ToUpper()'
+Description: 'Returns a new string in all uppercase characters based on the passed string.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.ToUpper()`** method is a method in C# that takes a string and returns a copy of this string converted to uppercase. Numbers and symbols are not changed.
+
+## Syntax
+
+```pseudo
+somestring.ToUpper()
+```
+
+> **Note:** This method will not change the input string.
+
+## Example
```

Comment:
Let's actually just keep the codebyte example here because the method is relatively straightforward. One example will suffice